kms/api-client/asymmetric_test.py

```diff
@@ -16,135 +16,116 @@
 from os import environ
 from time import sleep
 
+import asymmetric as sample
```

Comment:
The other tests in this folder all simply import the module under test. Is there a reason to rename it as sample for this one? If not, please just leave the name as asymmetric.

Reply:
I can't remember what the logic was behind this, but it probably made more sense before a refactor. Fixed